packages/pass-style/src/types.js

```diff
@@ -74,7 +105,14 @@ export {};
  */
 
 /**
- * @typedef {Passable} RemotableObject
+ * @template {string} S pass style
+ * @template {InterfaceSpec} I interface tag
+ * @typedef {PassStyled<S> & {[Symbol.toStringTag]: I}} TaggedRecord
```

Comment:
This use of "Tagged" is different from the `tagged` passStyle, right? That seems like a source of confusion.

Reply:
Different, yes. This is as in "toString**Tag**".  Naming suggestions welcomed

Reply:
I see.... :thinking:

Reply:
"PassStyled" seems like a good name... do we need both types, or can they be collapsed together?

Reply:
I wasn't sure. I thought I saw an object with the `[PASS_STYLE]` property and not the `toStringTag` or vice-versa. Are they supposed to be always present together?

Reply:
> Are they supposed to be always present together?

Yes, AFAIK. A Tagged has own `[PASS_STYLE]: "tagged", [Symbol.toStringTag]: $tag`, and a Remotable has a prototype chain in which the penultimate object has own `[PASS_STYLE]: "remotable", [Symbol.toStringTag]: $iface` (where both `$tag` and `$iface` must be strings, and the latter must either be "Remotable" or start with "Alleged: " or "DebugName: ").

Reply:
Thanks! 1b30d600cf5084b061590c12ca39294328bb69c6